sdk/sdk-api/src/main/java/mil/emp3/api/WMS.java

```diff
@@ -18,6 +18,7 @@
 
     final private IStorageManager storageManager = ManagerFactory.getInstance().getStorageManager();
 
+    private String serviceURL = null;
```

Comment:
Why do you need this? This is available in the base class. Also why not make all the members that are arguments to the constructor as final, unless you expect some one to modify those later.